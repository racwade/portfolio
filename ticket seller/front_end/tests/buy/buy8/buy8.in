login
test_buy
buy
test_sell
Empty
logout