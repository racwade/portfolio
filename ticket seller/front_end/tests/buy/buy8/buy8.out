welcome to the ticket seller system
enter username:
logged in as standard_buy
enter seller's username:
enter event name:
error: no remaining tickets
bye